src/sqlstore/timeline.py

```diff
@@ -0,0 +1,234 @@
+from sqlalchemy import Column, Integer, String, BigInteger, Boolean, ForeignKey, DateTime, PickleType, Identity
+from sqlalchemy.sql import func
+from src.sqlstore.db import Base
+
+
+class SQLTimeline(Base):
+    __tablename__ = "match_timeline"
+
+    platformId = Column(String(7), primary_key=True)
+    gameId = Column(BigInteger, primary_key=True)
+    frameInterval = Column(Integer)
+    # frameIds = Column(Integer, ForeignKey("match_timeline_frame.frameId"), nullable=False)
+    timeCreated = Column("timeCreated", DateTime(timezone=True), server_default=func.now())
+    lastUpdate = Column("lastUpdate", DateTime(timezone=True), onupdate=func.now())
+
+    def __init__(self, platformId: str, gameId: int, frameInterval: int):
+        self.platformId = platformId
+        self.gameId = gameId
+        self.frameInterval = frameInterval
+
+    def __repr__(self):
+        return f"{self.platformId}_{self.gameId} with interval {self.frameInterval}ms"
+
+
+class SQLTimelineFrame(Base):
+    __tablename__ = "match_timeline_frame"
+
+    platformId = Column(String(7), primary_key=True)
+    gameId = Column(BigInteger, primary_key=True)
+    frameId = Column(Integer, primary_key=True)     # this is not present in the data and needs to be calculated
+    # eventIds = Column(Integer, ForeignKey("match_timeline_event.eventId"))  # TODO: is this relevant here?
```

Comment:
I am not quite sure with this one. The foreignKey should probably be on the child and point to the parent here. TimeLineEvent -> timelineFrame

maybe use relationship() function. Foreign key on TimelineEvent and to indicate which frame it belongs to and on TimelineFrame use relationship() to create a collection of events for each single frame